src/components/MuiVirtualizedTable/MuiVirtualizedTable.js

```diff
@@ -60,6 +61,11 @@ const defaultStyles = {
     },
     rowBackgroundDark: {},
     rowBackgroundLight: {},
+    cellTooltip: {
+        maxWidth: '1260px',
+        fontSize: '0.9rem',
```

Comment:
is there any way to avoid hard coded values ? mui theme values maybe ?
It makes it look a bit weird on very big screen because of that